Show how to split on a composite primary key

rmloveland opened the pull request:
Fixes #2769

nstewart commented:
Thanks LGTM

RaduBerinde commented:
Maybe worth mentioning that you can also split at prefix of the PK columns. In this case, you could split at /3 (which means that there is a split before all keys that start with `3`)

cockroach-teamcity commented:
http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/ab6b38d494266a1e2ae470f479d03a58325dbaff/

rmloveland commented:
> Maybe worth mentioning that you can also split at prefix of the PK columns. In this case, you could split at /3 (which means that there is a split before all keys that start with 3)

Fixed in latest push - thanks!

jseldess commented on v2.0/split-at.md:
```diff
@@ -103,6 +103,87 @@ the ranges that store tables or indexes:
+(4 rows)
+~~~
+
+Alternatively, you could instead split at a prefix of the primary key columns.  For example, to add a split before all keys that start with `3`, run:
```
nits: Don't need both `alternatively` and `instead`. I'd remove `instead`. Also, use just one space following the period.

rmloveland replied:
Thanks, fixed.

jseldess commented on v2.1/split-at.md:
```diff
@@ -103,6 +103,87 @@ the ranges that store tables or indexes:
+(4 rows)
+~~~
+
+Alternatively, you could instead split at a prefix of the primary key columns.  For example, to add a split before all keys that start with `3`, run:
```
Same as above.

rmloveland replied:
Fixed.

cockroach-teamcity commented:
http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/772b74dd541b106a3ca133caa55c8514d06f8d7a/
